src/main/java/com/example/bookshop/dto/BookDto.java

```diff
@@ -0,0 +1,23 @@
+package com.example.bookshop.dto;
+
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class BookDto {
+    private Long id;
+
+    private String title;
```

Comment:
let's make use of the features new Java possesses - such classes should be records